doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,66 @@
+Traffic Shaping
+===============
+
+The Linux offers a very rich set of tools for managing and manipulating the transmission of packets.
+Those tools can be used to manage and shape traffic on SoftwareContainer.
+
+The Linux network routing, firewalling and traffic control subsystem is very powerful and flexible
+and have grown to be a very mature stack of tools. To glance those solutions the linux
+documentation project would be very good resource to start. Please check
+`TLDP <http://www.tldp.org/HOWTO/html_single/Traffic-Control-HOWTO/#intro>`_
+for having understanding of the concept. `Linux Advanced Routing & Traffic Control <http://lartc.org/howto/>`_
+is another good source that presents deep information about the topic.
+
+In spite of there are many ways of shaping the traffic, what we describe here is to give brief information
+with examples about one particular way of shaping network traffic of a container to prevent any possible
```

Comment:
Are we absolutely certain this will prevent "any possible" network starvation scenario? Ie, do we protect against _all attacks_" kind of? ;)